src/components/Cell.tsx

```diff
@@ -3,19 +3,17 @@ import waves from '../assets/wave.png';
 
 export interface CellInterface {
     shipName: string | null;
-    row: number;
-    column: number;
+    row?: number;
+    column?: number;
```

Comment:
Nit - convention is to put optional values on the bottom

Reply:
Fixed.